gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/oauth/OAuthPage.kt

```diff
@@ -106,56 +115,81 @@ internal fun OauthPage(
     uiState: OAuthUiState,
     email: Email,
     onStartOAuthClicked: () -> Unit,
-    onEmailAssociationCheckClicked: (String) -> Unit = {},
     modifier: Modifier = Modifier,
+    onEmailAssociationCheckClicked: (String) -> Unit = {},
 ) {
+    val context = LocalContext.current
     GravatarTheme {
         Surface {
-            Box(
+            Column(
                 modifier = modifier
-                    .fillMaxWidth()
-                    .height(DEFAULT_PAGE_HEIGHT),
+                    .padding(horizontal = 16.dp)
+                    .fillMaxWidth(),
             ) {
+                Text(
+                    text = stringResource(R.string.gravatar_qe_oauth_page_title),
+                    style = MaterialTheme.typography.headlineMedium.copy(
+                        fontWeight = FontWeight.Black,
+                        fontSize = 32.sp,
+                        letterSpacing = 0.4.sp,
+                    ),
+                    modifier = Modifier.padding(top = 10.dp),
+                )
+                QESectionMessage(
+                    message = stringResource(R.string.gravatar_qe_oauth_page_message, context.appName),
```

Comment:
⛏ I feel like we are missing a slight padding here:

```suggestion
                    message = stringResource(R.string.gravatar_qe_oauth_page_message, context.appName),
                    modifier = Modifier.padding(top = 4.dp),
```

<img width="350" alt="image" src="https://github.com/user-attachments/assets/f959c4d3-fb3a-475b-a800-f96ce4e50476" />

Reply:
You're right! Done.